.vscode/settings.json

```diff
@@ -31,7 +31,7 @@
   },
   "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
This is from the latest version of vscode's eslint, so threw this in here since other people are definitely seeing the same diff after extension/editor update.